lib/mpl_toolkits/axes_grid1/inset_locator.py

```diff
@@ -420,14 +435,27 @@ def inset_axes(parent_axes, width, height, loc=1,
             'center'       : 10
 
     bbox_to_anchor : tuple or `matplotlib.transforms.BboxBase`, optional
-        Bbox that the inset axes will be anchored. Can be a tuple of
-        [left, bottom, width, height], or a tuple of [left, bottom].
+        Bbox that the inset axes will be anchored to. If None,
+        *parent_axes.bbox* is used. If a tuple, can be either
+        [left, bottom, width, height], or [left, bottom].
+        If *width* and/or *height* are given in relative units, the 2-tuple
```

Comment:
"If the kwargs *width* and *height* are specified ..." Otherwise not clear that you are not referring to the width and height just mentioned.